docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/README.md

```diff
@@ -1,98 +1,100 @@
+endpoint is not available at the moment, the service will retry to send the message. After several retries, if the 
+message is still not delivered, message forwarder will back up the message, and continue with the next message. 
 
-1. **in-order message delivery** : If only one message processor is picking messages from the particular store, in-order message delivery pattern is also achieved. Messages are delivered to the endpoint in a reliable manner keeping the order of the messages they were put to the store.
-
-2. **throttling** : Consider a legacy backend which can process messages only up to 100 TPS. This service is exposed to a system from which it gets message bursts, sometimes exceeding its limit of 100 TPS. To regulate the load we can use message store and forward pattern. System can store the message in the store in the speed it likes and message processor will forward the messages to the backend in a regulated way with its defined polling interval.
+You can find more about the Store Forward Connector from [here](https://github.com/wso2-ballerina/module-storeforward).
 
 ## What you'll build
 
-Let's consider a scenario where an incoming HTTP request is stored using *messageStore*. The message store is pointed
-to a queue of *ActiveMQ* broker. The caller of the HTTP request will receive an HTTP accept *202* response, if the message
-is stored successfully on the broker. Message forwarding task is initiated by another Ballerina service which polls messages
-from the above store and invoke configured backend HTTP service.
+To understand how to implement reliable delivery for your HTTP service, consider a scenario where you have an API 
+for managing orders at an online shop. When an order is received at the `/orders/new-order` endpoint, it will send the request to 
+`/sales/save-order` endpoint for persisting the request. When transferring the order requests, reliable delivery should be ensured, 
+as messages can fail to deliver to the backend order service due to various reasons such as difference of processing 
+speeds in services and endpoint failures. 
 
-This backend service is a test service which will print headers and the incoming message, and respond with a static payload.
+![Reliable delivery](../../../../assets/img/reliable-delivery.png)
 
-Upon receiving the response from the backend, user is allowed to perform any action upon the response as per the design
-of *messageStore* module. This action is passed as a Lambda (function pointer) to the message processor when it is created.
-For simplicity, in this guide it will just log the response from the backend to the Ballerina log file.
+<!-- INCLUDE_MD: ../../../../tutorial-prerequisites.md -->
+* A message broker to store messages in a queue. You can download and install Apache ActiveMQ from 
+[here](http://activemq.apache.org/getting-started.html).
 
-The following diagram illustrates the scenario in high level:
+## Get the code
```

Comment:
There is a way to insert this section automatically. This is similar to how you include prerequisites. Please check.